webrender/src/device.rs

```diff
@@ -1250,6 +1217,13 @@ impl Device {
         texture.width = 0;
         texture.height = 0;
         texture.layer_count = 0;
+        texture.id = 0;
```

Comment:
hmm, not sure I understand the semantics here: is it valid for `Texture` to have `id==0`, effectively making it a null/invalid texture? In this case having `Option<Texture>` anywhere would be confusing. Otherwise (and preferred), can we not allow `id == 0` here?

Reply:
Oops, that's just a bug - well spotted! That line was left there from a previous iteration of this patch I was working on. Indeed, it's not valid for a texture to have id == 0 until it has been deleted.